test/TokenFactoryBase.t.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
+
+import "forge-std/Test.sol";
+import "@contracts/TokenFactoryBase.sol";
+import "@contracts/BancorBondingCurve.sol";
+import "@contracts/Token.sol";
+import "openzeppelin/contracts/proxy/Clones.sol";
+
+contract TokenFactoryTest is Test {
+    TokenFactoryBase tokenFactory;
+    BancorBondingCurve bondingCurve;
+    Token token;
+    address user = address(0x1);
```

Comment:
0x1 address is special and usually has a bunch of assets dumped to it, use something higher